app/docs/0.10.x/plugin-development/distribution.md

```diff
@@ -1,74 +1,300 @@
 ---
-title: Plugin Development - Distribute your plugin
+title: Plugin Development - (un)install your plugin
```

Comment:
user experience: folks looking for "distribution" will probably be looking at "(un)install", yet people looking for uninstall will not quickly look for "distribution".

So I think, as it is, people will find what they are looking for quicker.